pkgs/fc/agent/fc/manage/s3users.py

```diff
@@ -65,27 +66,35 @@ def __init__(self, uid):
         self.uid = uid
 
     def update(self, previously_deleted=False):
+        def check_user_doesnt_exist(code, stdout, stderr):
+            return (
+                code == 22
+                and b"could not fetch user info: no user info saved" in stderr
```

Comment:
I don't think this has a strip() before getting the stderr passed and the real output has a `\n` in it:

```
>>> x = subprocess.Popen(["radosgw-admin", "user", "info", "--uid=asdf"], stderr=subprocess.PIPE, stdout=subprocess.PIPE)
>>>
>>> x.communicate()
(b'', b'could not fetch user info: no user info saved\n')
```

Reply:
This is actually okay, since this is matched with `in`, but we could improve the check by matching on the exact string + `.strip()` if you want.

Reply:
you're right. i was misreading this as an equals match.